sql/moz-fx-data-shared-prod/fenix_derived/new_profile_activation_v1/query.sql

```diff
@@ -0,0 +1,123 @@
+WITH client_first_seen AS (
+  SELECT
+    client_id,
+    sample_id,
+    normalized_channel,
+    app_build,
+    app_channel,
+    app_display_version,
+    device_manufacturer,
+    device_model,
+    isp,
+    locale,
+    city,
+    country,
+    MIN(first_seen_date) AS first_seen_date
+  FROM
+    mozdata.fenix.baseline_clients_first_seen
+  WHERE
+    submission_date = DATE_SUB(@submission_date, INTERVAL 6 DAY)
+  GROUP BY
+    1,
+    2,
+    3,
+    4,
+    5,
+    6,
+    7,
+    8,
+    9,
+    10,
+    11,
+    12
+),
+client_search AS (
+  SELECT
+    client_id,
+    SUM(search_count) AS search_count
+  FROM
+    `moz-fx-data-shared-prod.search_derived.mobile_search_clients_daily_v1`
+  JOIN
+    client_first_seen
+  USING
+    (client_id)
+  WHERE
+    submission_date
+    BETWEEN DATE_SUB(@submission_date, INTERVAL 3 DAY)
+    AND @submission_date
+    AND normalized_app_name = 'Fenix'
+    AND DATE_DIFF(submission_date, first_seen_date, DAY)
+    BETWEEN 4
+    AND 7
+  GROUP BY
+    1
+),
+dou AS (
+  SELECT
+    client_id,
+    submission_date,
+    ARRAY_LENGTH(
+      mozfun.bits28.to_dates(mozfun.bits28.range(days_seen_bits, -5, 6), submission_date)
+    ) AS days_2_7,
+  FROM
+    `mozdata.fenix.baseline_clients_last_seen`
+  WHERE
+    submission_date = @submission_date
+    AND date_diff(submission_date, first_seen_date, DAY) = 6
```

Comment:
Here we're restricting to just the clients that were first seen 6 days ago, right?

Reply:
yes, this table is based on activity during their first 7 days so looking at the bit array at day 7 I can get the data.